py/dml/ctree.py

```diff
@@ -187,6 +192,43 @@ def assert_comparable_types(site, expr1, expr2, equality):
         typ2 = realtype(expr2.ctype())
         raise EILLCOMP(site, expr1, typ1, expr2, typ2)
 
+class CoverityComplianceBlock:
```

Comment:
looks unused, am I missing something?

Reply:
It is unused -- this is a WIP, after all.

Reply:
And no, it doesn't work. Seems you have to get GCC to recognize the pragma.

Reply:
OK. So far everything except the compliance block stuff looks good to me.

Reply:
It should be noted that for my RAII work I've done a revamp of how switch statements are parsed and generated, and that will allow for a much prettier solution re. SubsequentCases than I have right now -- the reason I haven't ported it over to this PR was to make this PR non-breaking.

Reply:
Removed